packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -394,10 +400,12 @@ class DerivedData extends DisposableController with AutoDisposeControllerMixin {
     // Get class with max retained size.
     final ClassStats theClass;
     if (classes is SingleHeapClasses) {
-      final classStatsList = classes.filtered(_core.classFilter.value);
+      final classStatsList =
+          classes.filtered(_core.classFilter.value, _core.rootPackage);
```

Comment:
Style nit: consider adding trailing commas so `classes.filtered(...` is on the same line as the assignment.